incubator/java-openliberty/templates/kafka/README.md

```diff
@@ -25,15 +25,13 @@ Start docker compose with the following command:
 
 If you run `docker network list`, you should see a new network with the name of your project directory and the word `_default` appended. For example, `test-appsody-kafka_default`.
 
-Alternatively, if you want to connect to a Kafka broker elsewhere, edit `src/main/resources/META-INF/microprofile-config.properties` and set the value of the `mp.messaging.connector.liberty-kafka.bootstrap.servers` property to the host and port number of the your broker.
-
 ### 3. Run the Appsody application in the new network
 
-Your Appsody application must be run in the same network as Kafka.
+Your Appsody application must be run in the same network as Kafka. You must also supply the host and port number of the Kafka broker as an environment variable. 
 
 Run the application using the following command:
 
-```appsody run --network test-appsody-kafka_default```
+```appsody run --network test-appsody-kafka_default --docker-options "--env MP_MESSAGING_CONNECTOR_LIBERTY_KAFKA_BOOTSTRAP_SERVERS=kafka:9092"```
```

Comment:
Maybe instead of this complicated, quoted invocation we could just have it in the **microprofile-config.properties** file commented-out, and instruct them here to edit the file?

Reply:
When deploying to a Kube environment I'd expected folks to use the env var, as the broker address is likely to differ between environments. Therefore IMO it makes sense for folks to use the same approach at dev time.  We could offer both options though... I'll add it to the config file and mention in the README